lib/src/identifier_position.dart

```diff
@@ -1 +1 @@
-enum IdentifierPosition { prepend, append, dontDisplay }
+enum IdentifierPosition { prepend, prependUnSpace, append , appendUnSpace, dontDisplay }
```

Comment:
Hi, thanks for helping out! Can you please rename the enum values to a more english sounding name, though?
Maybe something like `prependWithoutSpace` and `appendWithoutSpace`?

Reply:
Thank, i'm changed it. 😇
Sorry, for my language. 😅